.github/workflows/test.yml

```diff
@@ -8,13 +8,17 @@ jobs:
       fail-fast: false
       matrix:
```

Comment:
We should also extend the pipeline to add the matrix for the django setup in multiple verssion.